ClinGen/clincoded
# Connects to #1064. Clicking criteria codes takes users to the correct evaluation sections in VCI.

**Steps to test:**
1. Start an Interpretation in the VCI.
2. Click on different criteria codes in the criteria bar and expect to be taken to the expected evaluation sections consisting of the targeted criteria among the different tabs and subtabs within the VCI.
